### PR TITLE
fix(DB/Gossip): Korfax missing text

### LINKED ISSUE #8571: Korfax, Champion of the Light - Dialogue option missing text
### What client do you play on?

enUS

### Faction

- [X] Alliance
- [ ] Horde

### Content Phase:

- [ ] Generic
- [ ] 1-19
- [ ] 20-29
- [ ] 30-39
- [ ] 40-49
- [X] 50-59

### Current Behaviour
Original report: https://github.com/chromiecraft/chromiecraft/issues/2120

When you talk to the NPC [Korfax ](https://classic.wowhead.com/npc=16112/korfax-champion-of-the-light), he has 2 dialogue options alongside the repeatable quest. 1st option is "What is Dreadnaught armor?"  which, when you select, only says "Greetings, [player name]". On that screen there's another dialogue option "Continue please." which also says only "Greetings, [player name]". And after that there's a 3rd screen which says the same thing.

### Expected Blizzlike Behaviour

For the NPC to give out info about the armor.

Testers note:
The text can be found on [Wowpedia](https://wowpedia.fandom.com/wiki/Korfax), even though it is for wow classic, it should be the same

>What is Dreadnaught armor?
The first suit of Dreadnaught armor was made exclusively for the Scarlet Highlord Mograine. Whilst donning the full suit, the Highlord looked himself to be a demon from the depths of the Nether.
Aye, 'twas quite a sight to behold as Mograine would wade into a hundred Scourge, Ashbringer held high, and come out on the other side in a cloud of dust and flames - enemies left in ashes beneath his feet.

>Continue please.
It was not until after his controversial death that the knights and warriors of Lordaeron would wear similarly fashioned suits in honor of their fallen hero. While the death of Mograine was a great victory for Kel'Thuzad, the resulting martrydom would be felt until the end of the war.
Where one did fall, many would rise...

### Source

_No response_

### Steps to reproduce the problem

1. .go c id 16112
2. Talk and take second option

### Extra Notes

_No response_

### AC rev. hash/commit

https://github.com/chromiecraft/azerothcore-wotlk/commit/e9c71c8ebefc067e3119a2a1a63b74c3239cc55c

### Operating system

Ubuntu 20.04

### Modules

- [mod-ah-bot](https://github.com/azerothcore/mod-ah-bot)
- [mod-cfbg](https://github.com/azerothcore/mod-cfbg)
- [mod-chromie-xp](https://github.com/azerothcore/mod-chromie-xp)
- [mod-desertion-warnings](https://github.com/azerothcore/mod-desertion-warnings)
- [mod-duel-reset](https://github.com/azerothcore/mod-duel-reset)
- [mod-eluna-lua-engine](https://github.com/azerothcore/mod-eluna-lua-engine)
- [mod-ip-tracker](https://github.com/azerothcore/mod-ip-tracker)
- [mod-low-level-arena](https://github.com/azerothcore/mod-low-level-arena)
- [mod-multi-client-check](https://github.com/azerothcore/mod-multi-client-check)
- [mod-pvp-titles](https://github.com/azerothcore/mod-pvp-titles)
- [mod-pvpstats-announcer](https://github.com/azerothcore/mod-pvpstats-announcer)
- [mod-queue-list-cache](https://github.com/azerothcore/mod-queue-list-cache)
- [mod-server-auto-shutdown](https://github.com/azerothcore/mod-server-auto-shutdown)
- [lua-carbon-copy](https://github.com/55Honey/Acore_CarbonCopy)
- [lua-custom-worldboss](https://github.com/55Honey/Acore_CustomWorldboss)
- [lua-level-up-reward](https://github.com/55Honey/Acore_LevelUpReward)
- [lua-recruit-a-friend](https://github.com/55Honey/Acore_RecruitAFriend)
- [lua-send-and-bind](https://github.com/55Honey/Acore_SendAndBind)
- [lua-temp-announcements](https://github.com/55Honey/Acore_TempAnnouncements)
- [lua-zonecheck](https://github.com/55Honey/acore_Zonecheck)

### Customizations

None

### Server

ChromieCraft

### PR DESCRIPTION
**Pull request by aradep**
## Changes Proposed:
Added gossip_menu entries linking to existing npc_text.

## Issues Addressed:

- Closes #8571
- Closes chromiecraft/chromiecraft/issues/2120

## Tests Performed:

image

## How to Test the Changes:

.npc add 16112
Talk and ask about Dreadnaught armor

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Comment by Maelthyrr** [comment]
Tested and work fine. Thanks :)

File: data/sql/updates/pending_db_world/rev_1651987971178548000.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+--
+DELETE FROM `gossip_menu` WHERE `MenuID` IN (7106,7107);
+INSERT INTO `gossip_menu` VALUES (7106, 8363), (7107, 8364);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1651987971178548000.sql:3]: Sniffed?

Comment (aradep) [reply data/sql/updates/pending_db_world/rev_1651987971178548000.sql:3]: Kind of. Korfax has gossip_menu_options with ActionMenuID=7106/7107, but there is no gossip menu with those IDs so it breaks and shows default text. All I did was add the gossip_menu entries it was looking for and linked them to existing npc_text.

Comment (Kitzunu) [reply data/sql/updates/pending_db_world/rev_1651987971178548000.sql:3]: ok

**Review by Kitzunu — APPROVED** [review]

**Pull request merged**